packages/components/src/validated-form-controls/components/stories/checkbox-control.story.tsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * External dependencies
+ */
+import type { StoryObj, Meta } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import { ValidatedCheckboxControl } from '../checkbox-control';
+import { formDecorator } from './story-utils';
+
+const meta: Meta< typeof ValidatedCheckboxControl > = {
+	title: 'Components (Private)/Validated Form Controls/ValidatedCheckboxControl',
```

Comment:
```suggestion
	title: 'Components (Experimental)/Validated Form Controls/ValidatedCheckboxControl',
```

How about we move these all to the experimental components section, and add a "private" tag like the other locked components?

https://github.com/WordPress/gutenberg/blob/8189c9304bd6749f8ef43aa446e27e34eed245e5/packages/components/src/menu/stories/index.story.tsx#L58

Reply:
Addressed with 35c8eb47f49eca8d9b9ca2363a765dd19edd8e33